.github/scripts/update-platform-executables.js

```diff
@@ -47,62 +56,86 @@ const updatePlatformExecutables = async () => {
     const assetName = process.env.ASSET_NAME;
     const file = process.env.FILE;
     const overwrite = process.env.OVERWRITE;
+    const dirname= process.env.DIRNAME;
 
-    const stat = fs.statSync(file);
-    if (!stat.isFile()) {
-      core.setFailed(`Skipping ${file}, since its not a file`);
-      return;
-    }
-    const file_size = stat.size;
-    const file_bytes = fs.readFileSync(file);
+    const iterateOverFile = async (file,assetName) => {
+      const stat = fs.statSync(file);
+      if (!stat.isFile()) {
+        core.setFailed(`Skipping ${file}, since its not a file`);
+        return;
+      }
+      const file_size = stat.size;
+      const file_bytes = fs.readFileSync(file);
 
-    // Check for duplicates.
-    const assets = await github.repos.listAssetsForRelease(
-      {
-        owner,
-        repo,
-        release_id: oldReleaseId
-      });
-    const duplicate_asset = assets.data.find(a => a.name === assetName)
-    if (duplicate_asset !== undefined) {
-      if (overwrite) {
-        core.debug(
-          `An asset called ${assetName} already exists in release ${tag} so we'll overwrite it.`
-        )
-        await github.repos.deleteReleaseAsset({
+      // Check for duplicates.
+      const assets = await github.repos.listAssetsForRelease(
+        {
           owner,
           repo,
-          asset_id: duplicate_asset.id
-        })
+          release_id: oldReleaseId
+        });
+      const duplicate_asset = assets.data.find(a => a.name === assetName)
```

Comment:
Any sample runs to validate duplicate_asset logic block?

Reply:
Yes, refer this: [Run 1](https://github.com/isha689/twilio-cli/actions/runs/3243019344/jobs/5317132671) -> uploaded assets successfully.
[Run 2](https://github.com/isha689/twilio-cli/actions/runs/3243369117/jobs/5317982263) -> uploaded the same assets again, because overwrite flag has been set in oclif-release workflow.